src/Pages/Admin/DepartmentManagement/DepartmentModal.tsx

```diff
@@ -236,10 +237,34 @@ const DepartmentModal = ({
           </span>
           {departmentInfo.positions &&
             departmentInfo.positions.map((position, idx) => (
-              <div key={idx} className="form-control col mb-2">
-                {position}
+              <div key={idx} className=" form-inline form-control mb-2">
+                <div
+                  contentEditable="true"
+                  onInput={(e) => {
+                    if (e.currentTarget.textContent) {
+                      position = e.currentTarget.textContent;
+                    }
+                  }}
+                  onBlur={() => {
+                    editPosition(
+                      position,
+                      idx,
+                      departmentInfo,
+                      setDepartmentInfo
+                    );
+                  }}
+                  // on finish change editPosition(
+                  //       position,
+                  //       idx,
+                  //       departmentInfo,
+                  //       setDepartmentInfo
+                  //     );
+                  className="col-md-10 align-self-center"
+                >
+                  {position}
+                </div>
```

Comment:
Don't add previous PR changes in new ones, remove them from this, as this will be merged from the other PR

Reply:
previous feature changes removed from this PR